static/src/stylesheets/module/email/_header.scss

```diff
@@ -62,3 +62,15 @@
     }
 }
 
+// PLAIN MODS
```

Comment:
The plain version of the component style modification (Ie: BEM modifier). Comment probably makes more sense in context when you can see there are a different modifications: https://github.com/guardian/frontend/blob/RTRT_Email-interactives-event/static/src/stylesheets/module/email/_header.scss